### PR TITLE
feat: use_cynkra_pkgdown()

### LINKED ISSUE #6: Implement use_cynkra_favicon()
so that it's easy to add a pkgdown icon.

Also `use_cynkra_pkgdown()` that sets up everything, maybe later.

### LINKED ISSUE #28: Simplify inclusion
The README mentions the need to add a custom `in_header` setting. The tidytemplate has a very similar description, though.

Can we automate this, even only with a `use_cynkratemplate()` function?

### PR DESCRIPTION
**Pull request by maelle**
Fix #6 
Fix #28 

TODO
- fix cynkratemplate usage in cynkrathis

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -1,17 +1,16 @@
 url: https://github.com/cynkra/cynkratemplate
-
```
Comment (maelle) [inline _pkgdown.yml:2]: this is the kind of changes that happen when using this function :sweat_smile:

File: inst/pkgdown/templates/in-header.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<script defer data-domain="{{#package}}{{name}}{{/package}}.cynkra.com" src="https://plausible.io/js/plausible.js"></script>
```
Comment (maelle) [inline inst/pkgdown/templates/in-header.html:1]: I tested this locally, it worked, on cynkratemplate and cynkrathis.

**Comment by krlmlr** [comment]
Thanks! Can you please apply it to dm, fledge, and other public R package repositories in our org?

**Comment by maelle** [comment]
yes, will do later this week!

**Pull request merged**